abs/replica_client.go

```diff
@@ -145,11 +146,13 @@ func (c *ReplicaClient) Init(ctx context.Context) (err error) {
 }
 
 // LTXFiles returns an iterator over all available LTX files.
-func (c *ReplicaClient) LTXFiles(ctx context.Context, level int, seek ltx.TXID) (ltx.FileIterator, error) {
+// When timestamp is non-zero (timestamp-based restore), accurate timestamps are read from metadata.
+// Otherwise, fast timestamps from blob creation time are used.
+func (c *ReplicaClient) LTXFiles(ctx context.Context, level int, seek ltx.TXID, timestamp time.Time) (ltx.FileIterator, error) {
```

Comment:
`timestamp` seems like it could be a `bool` rather than the actual timestamp. It doesn't seem like you use the actual value, do you?